### PR TITLE
Remove-EntraAdministrativeUnitMember

### PR DESCRIPTION
**Pull request by v-uansari**

### CODE REVIEW

File: module/Entra/AdditionalFunctions/Remove-EntraAdministrativeUnitMember.ps1 [hunk 1]
```diff
@@ -0,0 +1,82 @@
+function Remove-EntraAdministrativeUnitMember {
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Remove-EntraAdministrativeUnitMember.ps1:1]: Add license information at the top of this file

File: test/module/Entra/Remove-EntraAdministrativeUnitMember.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,41 @@
+BeforeAll {
```
Comment (KenitoInc) [inline test/module/Entra/Remove-EntraAdministrativeUnitMember.Tests.ps1:1]: Add license information at the top of this file

File: module/Entra/AdditionalFunctions/Remove-EntraAdministrativeUnitMember.ps1 (hunk 1 above)
Comment (v-pughosh) [reply module/Entra/AdditionalFunctions/Remove-EntraAdministrativeUnitMember.ps1:4]: done

File: test/module/Entra/Remove-EntraAdministrativeUnitMember.Tests.ps1 (hunk 2 above)
Comment (v-pughosh) [reply test/module/Entra/Remove-EntraAdministrativeUnitMember.Tests.ps1:4]: done

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**